db/migrations/20250115112007_add_feature_name_to_repo_configs.down.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
+ALTER TABLE repository_configurations DROP COLUMN feature_name;
+
+COMMIT;
```

Comment:
I'd add an index on this column, as we'll be querying it in the future (and might as well do it in the same migration that adds it).